skyportal/handlers/api/observation_plan.py

```diff
@@ -2008,7 +2031,7 @@ def observation_simsurvey(
         order = hp.nside2order(localization.nside)
         t = rasterize(localization.table, order)
 
-        if 'DISTMU' in t:
+        if 'DISTMU' in t.colnames:
```

Comment:
Maybe you also want to check if the other columns you use in the if clause are used, not just this one? (DISTSIGMA, DISTNORM)

Reply:
implemented.